resources/js/contest-voting/entry.coffee

```diff
@@ -61,11 +61,19 @@ export class Entry extends React.Component
       if @props.contest.show_votes
         div className: 'contest-voting-list__title contest-voting-list__title--show-votes',
           div className: 'contest-voting-list__votes-bar', style: { width: "#{relativeVotePercentage}%" }
-          div className: 'u-relative u-ellipsis-overflow', entry_title
+          div className: 'u-relative u-ellipsis-overflow',
+            a
```

Comment:
the extra layer of div isn't necessary, as long as you make the link `display: block`. to combine these "utility" styles into the title link, add lines like `.u-ellipsis-overflow();` into the styles

so u may end up with something like...
```less
&--link {
  .u-ellipsis-overflow();
  .u-relative();

  display: block;
  
  // ...
}
```

also, it seems `u-relative` may not be needed either? I'm not sure why it's here